zero-thrs-control/tests/modules/boilers/conftest.py

```diff
@@ -0,0 +1,59 @@
+from datetime import datetime, timedelta
+from pytest import fixture
+
+from thrs.control.modules.boilers import BoilersParameters, Tank, TanksController
+from thrs.input_output.modules.boilers import BoilersControlValues, BoilersSensorValues
+
+
+class TestTime:
+    def __init__(self, duration: timedelta = timedelta(seconds=1)):
+        self._time = datetime.now()
+        self._duration = duration
+
+    def time(self) -> datetime:
+        self._time += self._duration
+        return self._time
```

Comment:
Kinda weird to have time mutate when it's called. e.g. if multiple PIDs get the `time_fn` and each of them calls `time` you'd advance time multiple times in a single `control`